app/graphql/types/data_management_plan_type.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class DataManagementPlanType < BaseObject
+  implements DoiItem
+
+  def type
```

Comment:
What about adding a `organisations` field, that aggregates all the affiliations found in both creators and contributors and those affiliations from contributors HostingInstitution. In the vision of the DMP mentions that we want the organisations, with a field as such users of the API will have it right away.

Reply:
I would not make these kinds of changes only for DMPs. I want the GraphQL API to be as generic as possible.

Reply:
Would this mean that we would let the user to figure out what constitute an `organisation` and/or `dataset` out of all the metadata they get in query for DMP (basically `OverFetching`)?

Reply:
Of course not. Organizations and datasets are different fields, so they can decide what they want to fetch.